core/src/main/res/layout/mz_view_map.xml

```diff
@@ -11,6 +11,20 @@
       android:layout_height="match_parent"
       />
 
+  <com.mapzen.android.graphics.CompassView
+      android:id="@+id/mz_compass"
+      android:layout_width="@dimen/mz_compass_button_width"
+      android:layout_height="@dimen/mz_compass_button_height"
+      android:layout_alignParentLeft="true"
```

Comment:
We should align the compass to be parent right by default since this is how both GMaps and EraserMap work